astroquery/esasky/core.py

```diff
@@ -373,10 +374,15 @@ def get_maps(self, query_table_list, missions=__ALL_STRING,
                             download_dir,
                             cache))
                     break
-        if (len(sanitized_query_table_list) > 0):
-            log.info("Maps available at %s" % os.path.abspath(download_dir))
+
+        if all([maps[mission].count(None) == len(maps[mission])
+                for mission in maps]):
+            log.info("No maps got downloaded, check errors above.")
+
```

Comment:
We could add this check to get_images also

Reply:
Done